platforms/tktrex/backend/parsers/native.ts

```diff
@@ -1,40 +1,42 @@
 import { ParserFn } from '@shared/providers/parser.provider';
+import { NativeMetadata } from '@tktrex/shared/models/Metadata';
 import D from 'debug';
 import { HTMLSource } from '../lib/parser';
-import parseAuthor from './author';
-import parseHashtags from './hashtags';
-import parseMusic from './music';
-import parseMetrics from './numbers';
 
 const debug = D('parser:native');
 
-const parseNativeVideo: ParserFn<HTMLSource, any> = async (
+const parseNativeVideo: ParserFn<HTMLSource, NativeMetadata> = async (
   envelop,
   findings
 ) => {
   if (envelop.html.type !== 'native') {
     debug('entry is not "native" (%s)', envelop.html.type);
     return null;
   }
-  debug('processing native video entry: %s', envelop.html.href);
+  debug('processing native video entry: %s %O', envelop.html.href, findings);
 
-  const music = await parseMusic(envelop, findings);
-  const author = await parseAuthor(envelop, findings);
-  const metrics = await parseMetrics(envelop, findings);
-  const hashtags = await parseHashtags(envelop, findings);
-
-  // description is already available at this point!
-  // const description = await parseDescription(envelop, findings);
-  const description = findings.description;
+  const clientTime: any = envelop.html.clientTime;
+  const savingTime: any = envelop.html.savingTime;
+  const music = findings.music;
+  const author = findings.author;
+  const metrics = findings.metrics;
+  const hashtags = findings.hashtags;
+  const description = findings.description.description;
+  const baretext = findings.description.baretext;
 
   return {
+    id: envelop.html.id,
```

Comment:
this is not OK.. composing the `Metadata` in a parser is really outside the pattern. let's talk about this